app/controllers/users_controller.rb

```diff
@@ -1,29 +1,29 @@
 class UsersController < ApplicationController
   before_action :authenticate_user!
 
-  def edit
-  end
+  # def edit
+  # end
   
   def show
      @user = User.find(params[:id])
   end
 
-  # Only users and admins can update user's profile
-  def update
-    if current_user.update_attributes(user_params)
-      redirect_to edit_user_registration_path
-    end
-  end
+  # # Only users and admins can update user's profile
+  # def update
+  #   if current_user.update_attributes(user_params)
+  #     redirect_to edit_user_registration_path
+  #   end
+  # end
```

Comment:
Yeah that was just to make reversal easier in case testing found I had broken something. ;-)